packages/api/tests/lib/reconciliationReport/test-normalizeEvent.js

```diff
@@ -110,14 +115,89 @@ test('normalizeEvent correctly handles granuleIds and collectionIds if reportTyp
     reportType: 'Internal',
     granuleId: 'someGranuleId',
     collectionId: 'someCollectionId1',
+    provider: 'someProvider1',
   };
 
   const expected = {
-    ...omit(inputEvent, ['collectionId', 'granuleId']),
+    ...omit(inputEvent, ['collectionId', 'granuleId', 'provider']),
     granuleIds: ['someGranuleId'],
     collectionIds: ['someCollectionId1'],
+    providers: ['someProvider1'],
   };
 
   const actual = normalizeEvent(inputEvent);
   t.deepEqual(actual, expected);
 });
+
+test('normalizeEvent moves string on provider to array on providers', (t) => {
+  const inputEvent = {
+    systemBucket: 'systemBucket',
+    stackName: 'stackName',
+    startTimestamp: new Date().toISOString(),
+    endTimestamp: new Date().toISOString(),
+    reportType: 'Not Internal',
+    provider: 'someProvider',
+  };
+  const expect = {
+    ...inputEvent,
+    providers: ['someProvider'],
+  };
+  delete expect.provider;
+
+  const actual = normalizeEvent(inputEvent);
+  t.deepEqual(actual, expect);
+});
+
+test('normalizeEvent moves array on provider to providers', (t) => {
+  const inputEvent = {
+    systemBucket: 'systemBucket',
+    stackName: 'stackName',
+    startTimestamp: new Date().toISOString(),
+    endTimestamp: new Date().toISOString(),
+    reportType: 'Not Internal',
+    provider: ['provider1', 'provider2'],
+  };
+
+  const expect = {
+    ...inputEvent,
+    providers: ['provider1', 'provider2'],
+  };
+  delete expect.provider;
+
+  const actual = normalizeEvent(inputEvent);
+  t.deepEqual(actual, expect);
+});
+
+test('normalizeEvent throws error if providers and collectionIds are passed to non-Internal report', (t) => {
```

Comment:
description doesn't match the test and error message

Reply:
I updated it to use real examples and fix the description to use singular param-names.